apps/examples/webserver_test/readme.md

```diff
@@ -0,0 +1,36 @@
+# Webserver Test Application
+
+### Features
+
+- HTTPS (based on mbedTLS h/w accelerate)
+- GET/PUT/POST/DELETE method
+- Supports Keep-alive
+- Chunked Transfer
+- Trailer header
+
+### Build Configuration
+
+> security_hal_test_tz
+
+### Menuconfig Setup
+**Enable web server**
+> Networking support > Protocols > Webserver 
+
+> Application Configuration > Examples > Webserver example
+
+> Application Configuration > Examples > Webserver test
+
+## webserver_test usage :
+
+**webserver_test iterations method uri [options...]**
```

Comment:
How about to combine readme.txt and readme.md. For example it's better to add option explanation to here.

Reply:
Done